notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb

```diff
@@ -0,0 +1,544 @@
+{
```

Comment:
<p>Parse and concatenate mask values</p><p> </p><p>"feature important" --&gt; "feature importance"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/182/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>